packages/thumbprint-react/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 14.3.6 - 2021-06-24
+
+### Changed
+
+-   [Patch] Upgrade `react-day-picker` to 8.8.0
+-   [Patch] Upgrade `date-fns` peer dependency to 2.30.0
```

Comment:
It's technically not a peer dependency. Here's an overview of peer dependencies: https://nodejs.org/fa/blog/npm/peer-dependencies

```suggestion
-   [Patch] Upgrade `date-fns` to 2.30.0
```

Reply:
Updated!